params/config/sim_config.yaml

```diff
@@ -12,4 +12,7 @@ deterministic_size: True           # default: True
 # deterministic values rather than means of a random distribution
 # can be overridden by deterministic_arrival and deterministic_size
 #deterministic: True                # default: False
-run_duration: 100                  # default: 100
\ No newline at end of file
+run_duration: 100                  # default: 100
+
+# Optional: Trace file trace relative to the CWD. Please place trace in the working directory
```

Comment:
what does that mean? so it's in `project root/params/traces/default_trace.csv`, right? I'd remove the second sentence